Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PodcastGuru 2.0.3-beta4'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Podcast Guru
+    version: "2.0.3"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'PodcastGuru/65 CFNetwork/1404.0.5 Darwin/22.3.0'
+  os:
+    name: iOS
+    version: "16.3"
+    platform: ""
+  client:
+    type: mobile app
+    name: Podcast Guru
+    version: "65"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PodcastGuru 1.9.9-beta3'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Podcast Guru
+    version: "1.9.9"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcast Player/1.0 (phone;android RMX3195_11_A.18)'
+  os:
+    name: Android
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: Podcast Player
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcast Player/1.31 (Linux;Android 13) ExoPlayerLib/2.8.2'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: Podcast Player
+    version: "1.31"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcast Player/1.5'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Podcast Player
+    version: "1.5"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcast Player/1.0 (phone;ios Version 16.3.1 (Build 20D67))'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Podcast Player
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'PodcastRepublic/18.0 (Linux; U; Android 10;blueline/QP1A.190711.020.C3)'
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: Podcast Republic
+    version: "18.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'PodcastRepublic/18.0 (iPhone; CPU iPhone OS 14_7_1 like Mac OS X)'
+  os:
+    name: iOS
+    version: "14.7.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Podcast Republic
+    version: "18.0"
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcastly/1.4.1 (samsung; CPU OS android 11; en_us)'
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: Podcastly
+    version: "1.4.1"
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'CPU OS android 11'
```

Comment:
model: ""

Reply:
This required a negative lookahead to resolve.